tsk2/tsk2/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace tsk2
+{
+    class Program
+    {
+        static char[] alphabet = { '0','1', '2', '3', '4', '5', '6', '7', '8', '9',
+                         'A', 'B', 'C', 'D', 'E', 'F','G','H','I','J'};
+        const int minNumericBase = 2;
+        const int maxNumericBase = 20;
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.WriteLine("Enter an integer number");
+                int number = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine($"Enter a numeric base(a number from {minNumericBase} to {maxNumericBase})");
+                int numericBase = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Entered number in the new system");
+                Console.WriteLine(NumberToNewSystem(number, numericBase));
+            }
+            catch (ArgumentOutOfRangeException ex)
```

Comment:
catch(Exception ex)
{
   if(ex is ArgumentOutOfRangeException || ex is ArgumentException || ...)
   {
      Console.WriteLine(ex.Message);
   }
}